playbooks/adhoc/atomic_openshift_tutorial_reset.yml

```diff
@@ -0,0 +1,68 @@
+# This deletes *ALL* Docker images, and uninstalls OpenShift and
+# Atomic Enterprise RPMs.  It is primarily intended for use
+# with the tutorial as well as for developers to reset state.
+
+- hosts:
+    - OSEv3:children
+
+  sudo: yes
+
+  tasks:
+    - service: name={{ item }} state=stopped
+      with_items:
+        - openshift-master
+        - openshift-node
+        - openvswitch
+        - atomic-enterprise-master
+        - atomic-enterprise-node
```

Comment:
@sdodson  Based on https://github.com/sdodson/origin/blob/package-refactor, I have come up with following:

1. origin, AE and openshift
 /etc/origin same for all 
what about /var/lib/origin? would it stay same for  openshift and AE or would it be /var/lib/atomic-openshift?

2. origin rpms: 
origin, origin-master, origin-node, origin-sdn-ovs, tuned-profiles-origin-node 

AE and openshift rpms
        - atomic-openshift
        - atomic-openshift-master
        - atomic-openshift-node
        - atomic-openshift-sdn-ovs
        - tuned-profiles-atomic-openshift-node

3. origin service files:
origin-master, origin-node

AE and openshift services:
        - atomic-openshift-master
        - atomic-openshift-node

4. origin /etc/sysconfig
        - /etc/sysconfig/origin-master
        - /etc/sysconfig/origin-node

AE and openshift:
        - /etc/sysconfig/atomic-openshift-master
        - /etc/sysconfig/atomic-openshift-node

5. binaries:
origin: /bin/origin
AE: /bin/atomic-enterprise
openshift: /bin/openshift